libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.IEntityRecognizer.uischema

```diff
@@ -0,0 +1,3 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema"
+}
```

Comment:
Do you actually use the interface schema types as a default for implementations?  I suspect you don't because there is really nothing you know about the specifics of implementations. If you look at the schemas for these you will see a $role of interface and implementations have a $role of implements(<interface>).

Reply:
I do not think we have a need for that, although maybe something to consider in the future to help manage duplication.

Reply:
The problem is there is no contract for the properties of implementations.  I would just remove these .uischema.  There is an opportunity for a default when people do "extends" like most of the conditions which extend OnCondition which has properties on it.

---
In reply to: [463191786](https://github.com/microsoft/botbuilder-dotnet/pull/4341#discussion_r463191786) [](ancestors = 463191786)

Reply:
That's fine with me. I just ran a script that created a uischema for every component. I'll remove the empty ones.